uwuno/src/main/java/com/learning/uwuno/services/gameService.java

```diff
@@ -17,7 +17,7 @@
     // Singleton for holding anything we need everywhere, should handle all business logic
 
     // Variables
-    private ArrayList<room> roomList = new ArrayList<room>();
+    private final ArrayList<room> roomList = new ArrayList<>();
```

Comment:
Why was this changed to final, if I understand correctly final == c++ const, and we definitely need to change this over the lifetime of the application.